app/Models/ConfigurationSetter.php

```diff
@@ -1,429 +0,0 @@
-	 * Return if the given key is supported by this setter.
-	 * @param string $key the key to test.
-	 * @return boolean true if the key is supported, false otherwise.
-	 */
-	public function support($key) {
-		$name_setter = '_' . $key;
-		return is_callable(array($this, $name_setter));
-	}
-
-	/**
-	 * Set the given key in data with the current value.
-	 * @param array $data an array containing the list of all configuration data.
-	 * @param string $key the key to update.
-	 * @param mixed $value the value to set.
-	 */
-	public function handle(&$data, $key, $value) {
-		$name_setter = '_' . $key;
-		call_user_func_array(array($this, $name_setter), array(&$data, $value));
-	}
-
-	/**
-	 * A helper to set boolean values.
-	 *
-	 * @param mixed $value the tested value.
-	 * @return boolean true if value is true and different from no, false else.
-	 */
-	private function handleBool($value) {
-		return ((bool)$value) && $value !== 'no';
-	}
-
-	/**
-	 * The (long) list of setters for user configuration.
-	 */
-	private function _apiPasswordHash(&$data, $value) {
-		$data['apiPasswordHash'] = ctype_graph($value) && (strlen($value) >= 60) ? $value : '';
-	}
-
-	private function _content_width(&$data, $value) {
-		$value = strtolower($value);
-		if (!in_array($value, array('thin', 'medium', 'large', 'no_limit'))) {
-			$value = 'thin';
-		}
-
-		$data['content_width'] = $value;
-	}
-
-	private function _default_state(&$data, $value) {
-		$data['default_state'] = (int)$value;
-	}
-
-	private function _default_view(&$data, $value) {
-		switch ($value) {
-		case 'all':
-			$data['default_view'] = $value;
-			$data['default_state'] = (FreshRSS_Entry::STATE_READ + FreshRSS_Entry::STATE_NOT_READ);
-			break;
-		case 'adaptive':
-		case 'unread':
-		default:
-			$data['default_view'] = $value;
-			$data['default_state'] = FreshRSS_Entry::STATE_NOT_READ;
-		}
-	}
-
-	// It works for system config too!
-	private function _extensions_enabled(&$data, $value) {
-		if (!is_array($value)) {
-			$value = array($value);
-		}
-		$data['extensions_enabled'] = $value;
-	}
-
-	private function _html5_notif_timeout(&$data, $value) {
-		$value = intval($value);
-		$data['html5_notif_timeout'] = $value >= 0 ? $value : 0;
-	}
-
-	// It works for system config too!
-	private function _language(&$data, $value) {
-		$value = strtolower($value);
-		$languages = Minz_Translate::availableLanguages();
-		if (!in_array($value, $languages)) {
-			$value = 'en';
-		}
-		$data['language'] = $value;
-	}
-
-	private function _passwordHash(&$data, $value) {
-		$data['passwordHash'] = ctype_graph($value) && (strlen($value) >= 60) ? $value : '';
-	}
-
-	private function _posts_per_page(&$data, $value) {
-		$value = intval($value);
-		$data['posts_per_page'] = $value > 0 ? $value : 10;
-	}
-
-	private function _queries(&$data, $values) {
-		$data['queries'] = array();
-		foreach ($values as $value) {
-			if ($value instanceof FreshRSS_UserQuery) {
-				$data['queries'][] = $value->toArray();
```

Comment:
This magic call was forgotten. Fixed in https://github.com/FreshRSS/FreshRSS/pull/5318